wiki/component-development.md

```diff
@@ -87,10 +87,10 @@ If a component has subcomponents (`<OuiToolBar>` and `<OuiToolBarSearch>`), tigh
 Refer to the [SASS page][sass] of our documentation site for a guide to writing styles.
 
 [component-design]: component-design.md
-[docs]: https://elastic.github.io/eui/
+[docs]: https://oui.opensearch.org/1.0/#/
```

Comment:
Should this include the "1.0/#/" part? Maybe just use the domain?

Reply:
Yea I will change the URL to only include the domain